src/zoteroOverlay.jsx

```diff
@@ -401,6 +402,29 @@ const zoteroOverlay = {
         }
     },
 
+    /**
+     * Set the URL value of a Zotero item with the value of it's P953 wikidata element statement 
+     */
+    fetchOpenAccessUrls: async function(menuName) {
+        const items = await this.getSelectedItems(menuName);
+
+        for (const item of items) {
+            // get the QID of the item
+            const qid = item.getPID('QID');
+
+            debug('QID : ' + qid);
+
+            // fetch the value of the P953 of the element
+            const OpenAccessUrl = await Wikidata.getProperties(qid, 'openAccessUrl');
```

Comment:
I have a hard time understanding how to fetch the value of a WD property with Cita's methods.
I might try to do this with the [hub](https://hub.toolforge.org/).